src/main/util/file_util.py

```diff
@@ -0,0 +1,147 @@
+import os
+import shutil
+
+import pandas as pd
+
+from src.main.util.consts import ACTIVITY_TRACKER_FILE_NAME, FILE_SYSTEM_ITEM, DATA_FOLDER_WITH_ATI, \
+    DATA_FOLDER_WITHOUT_ATI, ENCODING, LANGUAGE
+
+
+def remove_slash(path: str):
+    if path[-1] == '/':
+        path = path[:-1]
+    return path
+
+
+def add_slash(path: str):
+    if path[-1] != '/':
+        path += '/'
+    return path
+
+
+def get_file_name_from_path(file_path: str, with_extension=True):
+    file_path = remove_slash(file_path)
+    file_name = file_path.split('/')[-1]
+    if not with_extension:
+        file_name = file_name.split('.')[0]
+    return file_name
+
+
+def get_extension_from_file(file: str):
+    return file.split(".")[-1]
```

Comment:
Please use only ' symbols for strings (or " of course). But is better to use the ' symbol. It is necessary for a uniform code style. You can see for example https://stackoverflow.com/questions/628657/difference-between-the-use-of-double-quote-and-quotes-in-python